### PR TITLE
Duplicate Chef unit tests in Delivery.

### PR DESCRIPTION
**Pull request by tylercloke**

### CODE REVIEW

**Comment by stevendanna** [comment]
Heya, I know they are autogenerated, but can we clean up the copyright notices?

**Comment by tylercloke** [comment]
@stevendanna Of course. Thanks.

**Comment by tylercloke** [comment]
@coderanger I had duplicate PRs / branches. Was just closing the other one to get rid of duplicate. Posting your original comment here for reference:

https://github.com/chef/chef/pull/4109#issuecomment-151305262

**Comment by coderanger** [comment]
As mentioned on the mailing list :-1: :-1: :-1: :-1: :-1: :-1: :-1: until there is some kind of story for how non-employees interact with Delivery.

File: .delivery/config.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "version": "2",
+  "build_cookbook": {
+    "path": ".delivery/build-cookbook",
+    "name": "build-cookbook"
+  },
+    "skip_phases": [
+	"lint",
+	"syntax",
```
Comment (afiune) [inline .delivery/config.json]: Found the problem right here.. fixing it now! :smile:

**Comment by tylercloke** [comment]
@coderanger Hey, I'm developing in this branch, and this branch will not affect other PRs / the Chef project at large until it is merged. Let's plan on chatting about this at the Thursday developers meeting.

**Comment by coderanger** [comment]
@tylercloke Everyone that watches this repo is currently getting emails due to the bot, so maybe at least turn that off?

**Comment by lamont-granquist** [comment]
lol @chef-supermarket

**Comment by btm** [comment]
I'm closing this since we're doing something else.

**Pull request closed without merging**